### DISCUSSION TITLE
Windows 11 running slow on Mac M1 Pro

### DISCUSSION THREAD

**Discussion by VVgamer135** [utmapp/UTM#3610] [UTM Help (macOS)] [answered]
Hello, I am a new user of UTM. 
I finally got a virtual machine of Windows 11 ARM64 working (not and emulation), and everything is working well. But there is an annoying performance issue with the Cpu/ clock speed. I notice that the 'default' setting for the CPU only has a clock speed of 1 GHz. While this number is fine for basic program usage, the moment I try to open up Steam or any other semi performance-intensive program, the virtual machine slows to a crawl. Is there a way or special configuration that I can use to maximize the performance of my windows 11 virtual machine? 

Config
•UTM ver: 3.0.4 (46)
•Device: 2021 Apple MacBook Pro M1
•OS: MacOS 12.1

**Comment by osy** [comment, marked as answer]
The reported “speed” doesn’t mean anything. It’s slow because there’s no gpu acceleration. Most modern apps use some sort of gpu acceleration.

**Reply by VVgamer135** [reply]
Ohh, does UTM support any sort of GPU acceleration from the host os to the guest os?

**Reply by osy** [reply]
Not on windows

**Reply by VVgamer135** [reply]
oh dang, alright. so there's no other way to get more performance out of a windows 11 ARM virtual machine? I know you can increase the number of cores and the amount of RAM, but is that all you can do?

**Reply by onurdolu** [reply]
regardless of gpu, how to increase cpu speed ?
which one should be selected as a cpu emulation ?

**Reply by javitab** [reply]
What's been slow for you? Granted I'm on an M1 Pro and it auto assigned 8 cores, I'm able to watch 1080p YouTube videos, for example, without issue. How many cores/threads do you have if you open task manager where you see the "reported" speed?

**Reply by onurdolu** [reply]
<img width="830" alt="image" src="https://user-images.githubusercontent.com/1155700/161071632-b4d64d6b-be83-442c-8d62-adfdb78cc67d.png">

this is what's being selected for the windows 7 from the https://mac.getutm.app/gallery/windows-7

i have just M1, not pro.

should I write 4 core or 8 core on utm though ?
and which emulated card should be selected on display section for windows 7 ?

**Reply by onurdolu** [reply]
my question here was all about,
how to overcome the limitation on 1ghz cpu emulation  ?

**Comment by Cl1nkkk** [comment]
Set 8 cores and force multicore

**Comment by Brenonxcoder** [comment]
Maybe try installing the pro version. I had no problems with this system. The price is low in [this store](https://royalcdkeys.com/products/windows-11-pro-retail-cd-key), so it's worth trying.

**Reply by brian-xu** [reply]
Obvious astroturfing/promotion, should be removed

**Comment by WilliamHPeak** [comment]
Find something other than UTM....it's SLOW...

**Reply by realindiahotel** [reply]
It’s working perfectly for me on macOS 15.2 M4 Pro, not even the slightest bit slow

**Reply by djttam** [reply]
It's absolutely slow for me on an M4

**Comment by 0LEG0** [comment]
Using this tweak gave a significant acceleration of guest windows!
https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool
